lib/quantum/job_broadcaster.ex

```diff
@@ -18,24 +18,25 @@ defmodule Quantum.JobBroadcaster do
    * `jobs` - Array of `Quantum.Job`
 
   """
-  @spec start_link(GenServer.server(), [Job.t()]) :: GenServer.on_start()
-  def start_link(name, jobs) do
+  @spec start_link(GenServer.server(), [Job.t()], boolean) :: GenServer.on_start()
+  def start_link(name, jobs, debug_logging) do
     __MODULE__
-    |> GenStage.start_link(jobs, name: name)
+    |> GenStage.start_link({jobs, debug_logging}, name: name)
     |> Util.start_or_link()
   end
 
   @doc false
   @spec child_spec({GenServer.server(), [Job.t()]}) :: Supervisor.child_spec()
```

Comment:
You need to add `boolean` to this type spec.

`@spec child_spec({GenServer.server(), [Job.t()], boolean()}) :: Supervisor.child_spec()`

This is also the reason why the tests do not pass.